ddtrace/contrib/requests/__init__.py

```diff
@@ -33,5 +33,5 @@
 
 with require_modules(required_modules) as missing_modules:
     if not missing_modules:
-        from .patch import TracedSession, patch
-        __all__ = ['TracedSession', 'patch']
+        from .patch import TracedSession, patch, unpatch
+        __all__ = ['TracedSession', 'patch', 'unpatch']
```

Comment:
Is the deal to expose and document the `unpatch` method for this integration?

Reply:
No actually no, it has been exposed only for testing purposes. We probably don't want to suggest patching and unpatching at runtime. Anyway there is a plan to refactor the patching system, since we should start to introduce settings for each integration.